ui_tests/src/test/resources/com/saucedemo/account/signInPositive.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Sign In with valid data
+
+  @signIn
+  Scenario: Sign In with valid user
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+
+
+  @signIn
+  Scenario: Sign In with invalid password
+    Given I open start page
+    When I set 'performance_glitch_user' as username on Sign in page
+    And I set 'User123' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
```

Comment:
It's not a positive test case. Please explain

Reply:
negative testing ?

Reply:
@doniyorbobo The expected result of this test case is an error: "Epic sadface: Username and password do not match any user in this service". The expected result for a positive test case should be I am on Product page, i.e. I'm logged in

Reply:
@doniyorbobo The expected result of this test case is an error: "Epic sadface: Username and password do not match any user in this service". The expected result for a positive test case should be I am on Product page, i.e. I'm logged in